packages/mail/templates/cleaned_user_subscription_invitation.html

```diff
@@ -0,0 +1,155 @@
+                      folgendem Link die Newsletter wieder aktivieren:
+                    </p>
+                    <table
+                      width="100%"
+                      border="0"
+                      cellspacing="0"
+                      cellpadding="0"
+                    >
+                      <tr>
+                        <td>
+                          <table border="0" cellspacing="0" cellpadding="0">
+                            <tr>
+                              <td align="center" bgcolor="#3CAD00">
+                                <!-- TODO Link zum subscribe formular !!!! -->
+                                <a
+                                  href="http://eepurl.com/drh7uL"
+                                  style="font-size:20px;{{sg_font_style_sans_serif_regular}}color:#ffffff;text-decoration:none;border-radius:0;padding:15px 30px 15px 30px;border:1px solid #3CAD00;display:inline-block;"
+                                  >E-Mail-Newsletter reaktivieren</a
+                                >
+                              </td>
+                            </tr>
+                          </table>
+                        </td>
+                      </tr>
+                    </table>
+                    <p>
+                      Und falls es – was leider nicht ausgeschlossen werden kann
+                      – künftig zu weiteren Ausfällen kommen sollte, helfen wir
+                      Ihnen gerne weiter unter
+                      <a href="mailto:kontakt@republik.ch"
+                        >kontakt@republik.ch</a
+                      >.
+                    </p>
+                    <p>
+                      Wir wünschen Ihnen noch einen angenehmen Tag und weiterhin
+                      anregende Lektüre.
+                    </p>
+                    <p>Herzlich</p>
+                    <p>Ihre Crew der Republik</p>
+                    <p>
+                      PS: Möglicherweise ist das für Sie ja ganz gut, das Leben
+                      ohne Republik-Newsletter. Ja, vielleicht sogar gewollt,
+                      weil Sie sowieso immer direkt über unsere
+                      <a href="{{link_signin}}">Website</a>
+                      oder die
+                      <a href="{{link_faq}}#gibt-es-eine-app">Republik-App</a>
+                      auf die Inhalte zugreifen und die
+                      <a href="{{frontend_base_url}}/newsletter">Newsletter</a>
+                      Ihr Postfach eigentlich nur überflutet haben. Dann müssen
+                      Sie jetzt auch nichts weiter unternehmen und dürfen sich
+                      gerne wieder der Lektüre von Interessanterem als dieser
+                      Nachricht widmen. Bitte entschuldigen Sie die Störung.
+                    </p>
+                    <p>
+                      PPS: Kennen Sie schon unsere
+                      <a href="{{frontend_base_url}}/gebrauchsanleitung"
```

Comment:
Shorthand link from merge variables: `{{link_manual}}`

Reply:
Ah yes, searched for gebrauchsanleitung and didn't find it